.gitignore

```diff
@@ -59,3 +59,7 @@ typings/
 
 # next.js build output
 .next
+
+.nyc_output/
+
+dist
```

Comment:
Add a new line here at the end of the file

Reply:
Ok
Thanks